src/classes/CW_BriefApprovalProcess.cls

```diff
@@ -0,0 +1,167 @@
+/**
+* File Name   :    CW_BriefApprovalProcess .cls
+* Description :    B-106804 As a Marketing Ops I want the ability to submit a MarCom Brief
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 24/04/2014 Cristina P.	Intial
+*
+*/
+public with sharing class CW_BriefApprovalProcess 
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.